### ISSUE TITLE
Fix `declaration-property-value-no-unknown` false positives for calc() with mixed operations

### ISSUE THREAD

**Issue by ehoogeveen-medweb** [stylelint/stylelint#9066]
### What minimal example or steps are needed to reproduce the bug?

```css
.foo-5 {
  top: calc(0.3 * 100vh + 15px)
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "declaration-property-value-no-unknown": true
  }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0AzCEC0BGABMAOgHY44AuEADvDmAIYA2YAFFgKwUAeAlEQL5FFU6DACY8REuSo0GzLAAZ5ANwAWPQv0KC0mAMzjiZStTqMWi1TgDUONp3WbtwgCwHJxmWflJ9AKlsWanwChEKYrG5G0qbM3n4ByirWtuzcfCAANCAoAJb0MABytAC2cIgwHCUU+UhgAM51meAQhLkA5gggFRQQAE6kOLAotACu9AMEhr1jMHXUkyQk+CCwYPS0vbSkOS0YFL2UMP0AnhhKDCMwGISYI4QA1jcA7loIZNMwGRI4vF8aANxEJqQVo5NoAMT6xS2nTqpGO+XoOUIpFqLXaSGKACtGllYBRGogFjhlnCETAkSjltRlutSLNSMs-ksQGTEcjSBgQe0MHDaIQoBsoNSSSA6QzlsFCCBeEA)

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "17.2.0"
}
```

### What did you expect to happen?

No problems to be reported.

### What actually happened?

The following problems were reported;

```
18:8-32
    error
    Unexpected invalid math expression "calc(0.3 * 100vh + 15px)" for property "top"
    [(declaration-property-value-no-unknown)](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown)
```

### Do you have a proposal to fix the bug?

This regressed between v17.1.1 and v17.2.0.

**Comment by ybiquitous** [comment]
@ehoogeveen-medweb Thanks for the report. Now fixing.

**Comment by ehoogeveen-medweb** [comment]
Good to hear :) No rush from my end, the actual case that triggered the false positive in our code turned out to be unused (it was clamping a big calculation to a constant value, and had been that way for years).
